config/default.yml

```diff
@@ -4015,6 +4015,26 @@ Style/LineEndConcatenation:
   VersionAdded: '0.18'
   VersionChanged: '0.64'
 
+Style/MagicCommentFormat:
+  Description: 'Use a consistent style for magic comments.'
+  Enabled: pending
+  VersionAdded: '<<next>>'
+  EnforcedStyle: snake_case
+  SupportedStyles:
+    # `snake` will enforce the magic comment is written
+    # in snake case (words separated by underscores).
+    # Eg: froze_string_literal: true
+    - snake_case
+    # `kebab` will enforce the magic comment is written
+    # in kebab case (words separated by hyphens).
+    # Eg: froze-string-literal: true
+    - kebab_case
+  DirectiveCapitalization: lower
+  ValueCapitalization: ~
+  SupportedCapitalizations:
+    - lower
```

Comment:
I think I like `DirectiveCase` a bit better as `capitalization` and `lowercase` don't mix very well IMO. :-) Perhaps spelling out fully `lowercase` and `uppercase` would be best.

Reply:
I had `DirectiveCase` first but I found it confusing with snake case and kebab case. I'll change it to the full `lowercase` and `uppercase`, good suggestion!

Reply:
Updated!